vignettes/download_mail.Rmd

```diff
@@ -0,0 +1,182 @@
+---
+title: "Download Mod Mbox and Pipermail Mailing List Archives"
+output: 
+  html_document:
+    toc: true
+    number_sections: true
+vignette: >
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteIndexEntry{Download Mod Mbox Mailing List Archives}
+  %\VignetteEncoding{UTF-8}
+---
+
+
+```{r}
+rm(list = ls())
+seed <- 1
+set.seed(seed)
+
+# Load libraries
+  require(kaiaulu)
+  require(data.table)
+  require(yaml)
+  require(stringi)
+  require(XML)
+  require(httr)
+```
+
+
+# Introduction
+
+Mailing list data is stored in a variety of archives. See:
+- Mod Mbox: [Apache Geronimo](https://geronimo.apache.org/mailing-lists.html)).
+- Pipermail: [OpenSSL](https://mta.openssl.org/mailman/listinfo/).
+This notebook demonstrates how to download and refresh mailing list archives from Mod Mbox and Pipermail.
+
+# Pipermail
+
+## Mailing List Organization
+
+Mailing lists are typically organized by topic or purpose. For example, the [OpenSSL project](https://www.openssl.org/community/mailinglists.html) maintains several mailing lists, each serving a different group:
+
+- **openssl-announce**: For important announcements.
+- **openssl-commits**: For commit messages.
+- **openssl-project**: For project discussions.
+- **openssl-users**: For general user questions and discussions.
+
+Each mailing list maintains archives of past messages, often organized by month and year. These archives can be accessed and downloaded for analysis.
```

Comment:
This should be expanded. In addition to having mailing lists as different topics, each mailing list history may be split into multiple archives. See OpenSSL and create a example out of that, as I did with you on call.

Reply:
Can you please expand on this?

Reply:
In your OpenSSL mention, if you access the link in your notebook, you will see that the archive of a mailing list, for example -dev, may be incomplete. I believe the website even mentions one archive file is more complete than other. In short: A user may need to carefully examine each archive file rather than blindly choose any and go with it. Does that make sense? The choice of archive is contrained by Kaiaulu being able to parse it (pipermail / mod_mbox / something else in the future), and the time coverage. Sometimes it is more than time that is lacking, maybe one archive type will contain the "in_reply_to" information, whereas others may not. This should be cautioned on your notebook.

Reply:
Thank you for clarifying!